installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} version {product-version}, you can install a cluster on Microsoft Azure by using infrastructure that you provide.
+In {product-title} version 4.14, you can install a cluster on Microsoft Azure by using infrastructure that you provide.
```

Comment:
based on this fix, I understand that this feature is available only in 4.14. Not in the future versions. 
Can you confirm?

Reply:
Addressed